docs/UserGuide.md

```diff
@@ -74,13 +74,32 @@ Here is a sequence of example commands you can try:<br>
 
 ## Features
 
-**Notes about the command format:**<br>
+Excited about what Productiv can do for you? Here are the 4 main features of Productiv!
+1. [Dashboard](#dashboard) - Gives you an overview of your product’s development and upcoming events so that you can work better towards production deadlines.
+1. [Deliverable](#deliverable) - Helps you track your product’s deliverables so that you can work better towards meeting them.
+1. [Meeting](#meeting) - Helps you manage your product-related meetings	so that you can be clear on your meeting schedule.
+1. [Contact](#contact) - Helps you organise your developer or stakeholder contacts so that you can reference them easily.
 
-* Words in upper case are the parameters to be supplied by you for their respective fields.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter for the name field `n`, which can be used as `add n/Jason`.
 
-* Field-parameter pairs in square brackets are optional.<br>
-  e.g `n/NAME [p/PHONE]` can be used as `n/Jason p/98890112` or as `n/Jason`.
+**Listed below are the common components of a command:**<br>
+
+Component of Command | Description |
+---------------------|------------|
+Command word | The first word of a command. It is the action that you want to perform. |
+Prefix | The characters before the parameter. It is the short-form of the corresponding field. |
+Parameter |	The words following each prefix. It represents the value of the corresponding field. |
+
+For example, here's the breakdown of the command `add t/TITLE by/DEADLINE m/MILESTONE [c/CONTACTS] [d/DESCRIPTION]`:
+* Command word - `add`
+* Prefixes - `t/`, `by/`, `m/`, `c/`, `d/`
+* Parameters - `TITLE`, `DEADLINE`, `MILESTONE`, `CONTACTS`, `DESCRIPTION`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** Field-parameter pairs in square brackets are optional.
```

Comment:
Can consider to change to "Prefix-parameter pairs in square brackets are optional, i.e. `[c/CONTACTS]` and `[d/DESCRIPTION]` need not be present in the command." ?